recipes/guessmylt/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "guessmylt" %}
+{% set version = "0.2.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 30f9b300cc61c6f556605380bc4ba68d9eb3f3a5beb2f060cb69f9641f9c660c
+
+build:
+  number: 0
+  noarch: python
+  skip: True # [not win or osx]
+  entry_points:
+    - GUESSmyLT = GUESSmyLT.GUESSmyLT:main
+    - GUESSmyLT-example = GUESSmyLT.test.example:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+
+requirements:
+  host:
+    - python>3
+    - bcbio-gff ==0.6.4
+    - biopython ==1.67
+    - pip
+    - pysam >=0.13.0
+    - snakemake ==5.4.*
+    - trinity ==2.8.4
+    - busco ==3.0.2
+    - bowtie2 >=2
+  run:
+    - python>3
+    - git >=2.11.1
+    - bcbio-gff ==0.6.4
+    - biopython ==1.67
+    - pysam >=0.13.0
+    - snakemake ==5.4.*
+    - trinity ==2.8.4
+    - busco ==3.0.2
+    - bowtie2 >=2
+
+test:
+  imports:
+    - GUESSmyLT
+  commands:
+    - GUESSmyLT --help
+    - GUESSmyLT-example
+
+about:
+  home: https://github.com/NBISweden/GUESSmyLT
+  license: GPLv3
+  license_family: GPL
+  license_file: LICENSE
+  summary: Software to guess the RNA-Seq library type.
+  dev_url: https://github.com/NBISweden/GUESSmyLT
+
+extra:
+  recipe-maintainers:
+    - juke34
```

Comment:
Here in `extra` you should add your `skip-lints:` block

Something like:

```
skip-lints:
  - check_noarch
```

Reply:
Thx I will give a try

Reply:
my bad, try with `should_not_be_noarch_skip`

Reply:
Ok I get how it is supposed to work then, I will give a try

Reply:
It works fine for the first linting, but it doesn't work for the linting in the bioconda-test... Is there any extra tip to be aware of?

Reply:
adding `  skip-lints: #  The skip mechanism does not work in conjunction with noarch but I need to skip osx due to inconsistency in BUSCO
    - should_not_be_noarch_skip
    - should_not_be_noarch` works now. Thx